erpnext/selling/doctype/customer/customer.js

```diff
@@ -140,9 +140,53 @@ frappe.ui.form.on("Customer", {
 		var grid = cur_frm.get_field("sales_team").grid;
 		grid.set_column_disp("allocated_amount", false);
 		grid.set_column_disp("incentives", false);
+
+		if(frm.doc.__onload && frm.doc.__onload.dashboard_info) {
+			var company_wise_info = frm.doc.__onload.dashboard_info;
+			frm.set_value("total_monthly_sales", company_wise_info[0].billing_this_month);
```

Comment:
This throws an error if there are no sales for that customer

`Uncaught (in promise) TypeError: Cannot read property 'billing_this_month' of undefined`